opendj-ldap-toolkit/src/main/java/com/forgerock/opendj/ldap/tools/AddRate.java

```diff
@@ -435,7 +434,6 @@ int run(final String[] args) {
                     StringArgument.builder(MakeLDIF.OPTION_LONG_RESOURCE_PATH)
                             .shortIdentifier('r')
                             .description(INFO_ADDRATE_DESCRIPTION_RESOURCE_PATH.get())
-                            .docDescriptionSupplement(SUPPLEMENT_DESCRIPTION_RESOURCE_PATH.get())
```

Comment:
Is this change intentional?

Reply:
The change was intentional because the source file was deleted. But it is not relevant anymore, source files were reverted back.